.github/workflows/refapp-3x-login.yml

```diff
@@ -0,0 +1,38 @@
+name: RefApp 3.x Login
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  repository_dispatch:
+    types: [ qa ]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./qaframework-bdd-tests
```

Comment:
You probably don't need to set the working directory since you can directly run the tests after checking out the openmrs-test-3refapp module.